src/main/java/frc/robot/subsystems/GripperSystem.java

```diff
@@ -40,38 +42,11 @@ public void spin(double speed){
     rollerMotor.set(speed);
   }
 
-  /**
-   * Spins the gripper roller to intake
-   * sets speed to 0 to stop
-   **/
-  public CommandBase intake(){
-    return runEnd(
-      //run
-      () -> {
-        spin(ROLLER_SPEED);
-      },
-      //end
-      () -> {
-        spin(0);
-        
-        /** This logic changes the vision mode depending on whatever game piece has been grabbed */
-        if(checkForCube()) {
-          limelight.setPipeline(1);
-        } else if(checkForGamePiece()) {
-          limelight.setPipeline(0);
-        }
-
-      }
-
-    );
-  }
-
-
-  private boolean checkForGamePiece() {
+  public boolean checkForGamePiece() {
     return colorSensor.getIR() > GripperConstants.GAME_PIECE_IR_MINIMUM;
   }
 
-  private boolean checkForCube() {
+  public boolean checkForCube() {
     return checkForGamePiece() && colorSensor.getColor().blue > MINIMUM_BLUE_VALUE_FOR_CUBE;
   }
 
```

Comment:
would it be fitting to add a `checkForCone` method? this could be created using a combination of the cube check and game piece check methods, so it would largely be a convenience feature